openapi-integration-tests/src/test/java/io/ballerina/openapi/extension/build/BuildExtensionTests.java

```diff
@@ -47,18 +49,17 @@ public void setupDistributions() throws IOException {
     }
 
     @Test(description = "Check openapi build plugin in `bal build` command")
-    public void annotationWithOutBuildOption() throws IOException {
+    public void onlyBuildCommand() throws IOException {
```

Comment:
Shall we use more appropriate naming convention for the test cases? (This seems to be a negative test case to me). Better to refactor the also the other test case names in this class.